src/AppInstallerCommonCore/AppInstallerTelemetry.cpp

```diff
@@ -179,6 +179,8 @@ namespace AppInstaller::Logging
             packageVersion = Runtime::GetPackageVersion();
         }
 
+        Logging::SetActivityId();
```

Comment:
Why is this moved here?

Reply:
Earlier commit missed setting activity Id for winget-cli. I did that in our preview package, though. So, commiting it here and Instead of setting ActivityID at two different spots for COM and cli paths, I felt it's better to do at one spot, here.

Reply:
It is probably fine for now; we should just make it part of the object construction in the "make not global" PR.

Reply:
Sure. That is the plan with that pending code change.